specification/keyvault/Security.KeyVault.Secrets/routes.tsp

```diff
@@ -114,7 +114,14 @@ op getSecret is KeyVaultOperation<
      */
     @path("secret-version")
     @clientName("version", "go")
-    secretVersion?: string;
+    secretVersion: string;
```

Comment:
Version parameters like this one were intentionally made optional in https://github.com/Azure/azure-rest-api-specs/pull/34032 because the path segment can be left out in requests.
```suggestion
    secretVersion?: string;
```